src/query_parser/v2/parser.y

```diff
@@ -1141,28 +1094,20 @@ param_size(A) ::= ATTRIBUTE(B). {
   A.type = QT_PARAM_SIZE;
 }
 
-//For generic parameter (param_any) its `type` could be refined by other rules which may have more accurate semantics,
-// e.g., could know it should be numeric
-
-param_any(A) ::= ATTRIBUTE(B). {
-  A = B;
-  A.type = QT_PARAM_ANY;
-  A.inclusive = 1;
-}
-
-param_any(A) ::= LP ATTRIBUTE(B). {
-  A = B;
-  A.type = QT_PARAM_ANY;
-  A.inclusive = 0; // Could be relevant if type is refined
-}
-
-param_any(A) ::= TERM(B). {
-  A = B;
-  A.type = QT_TERM;
+param_num(A) ::= ATTRIBUTE(B). {
+    A = B;
+    A.type = QT_PARAM_NUMERIC;
+    A.inclusive = 1;
 }
```

Comment:
So absolutely no need for `param_any`? Nice!

Reply:
Sweet!